pkg/dcmtime/common.go

```diff
@@ -13,14 +13,14 @@ import (
 // Example: to test whether seconds should be included, you would:
```

Comment:
nit: it may be helpful to define is "included" better: e.g. A precision level is considered included if it is less precise than the Precision limit. For example, if the limit is PrecisionSeconds, PrecisionMinutes would be considred included but PrecisionMilliseconds would not be

Reply:
Updated, let me know if that works for you.

Reply:
LGTM, thank you!